### PR TITLE
Address Eliots initial comments

### PR DESCRIPTION
**Pull request by wkumari**
These set of changes address Eliot's initial ISE comments.

### CODE REVIEW

File: i-d/pdns-qof.xml [hunk 1]
```diff
@@ -482,11 +471,68 @@ ws              = *(
...
+    <reference anchor="DNSDB" target="https://api.dnsdb.info/">
+      <front>
+        <title>DNSDB API</title>
+        <author fullname="Farsight Security" />
+        <date year="2013" />
+      </front>
+    </reference>
+
+    <reference anchor="PDNSCERTAT"
+      target="http://www.centr.org/system/files/agenda/attachment/d4-papst-passive_dns.pdf">
```
Comment (aaronkaplan) [inline i-d/pdns-qof.xml:513]: That link seems broken!

Comment (adulau) [reply i-d/pdns-qof.xml:513]: If we cannot get the PDF back, we can safely remove the reference. As discussed, I checked on archive.org and there is archive of this presentation.

**Comment by adulau** [comment]
Thanks a lot for the PR. The changes make a lot of sense. I republished the I-D as 13 https://datatracker.ietf.org/doc/draft-dulaunoy-dnsop-passive-dns-cof/ and if there is more reviews or update, I'll do a 14 after.

**Comment by wkumari** [comment]
I replied to Eliot with a summary of changes, and also linking to the new
document / diff.

I also made the change suggested above — we actually said that the document
describes the format used by four implementations, but had listed 5, so
deleting one was easy :-)

W

**Pull request merged**